examples/example_1d.py

```diff
@@ -0,0 +1,38 @@
+from src.bayesian import run_one_1d_bayesian_optimization
+
+# fake data
+data = [{"x": 1, "y": 0},
+        {"x": 1.5, "y": 1.5},
+        {"x": 2, "y": 1},
+        {"x": 3, "y": -1}]
+
+mu, sigma, grid, s = run_one_1d_bayesian_optimization(data=data, limits={"x": (0, 4)})
+
+# plot
+import plotly.graph_objects as go
+
+traces = []
+traces.append(go.Scatter(x=grid, y=mu, mode='lines', name='mu'))
+traces.append(go.Scatter(
+    x=[s,s],
+    y=[-2,3],
+mode='lines',name="sugesstion"))
+traces.append(go.Scatter(x=grid, y=mu-sigma,
+    fill=None,
+    mode='lines',
+    line_color='indigo', name='mu+-sigma'
```

Comment:
You could use the ± symbol in the source code if you liked

Reply:
Also, it's not clear to me what sigma is meant to be. Is it the standard deviation? Some measure of variance? The 95th percentile?

Reply:
Ill add what they mean, I should add what they mean in the function description too